File: lec_004(13th oct)/array.cpp
#include<iostream>
#include<vector>

using namespace std;

void arr_input(vector<int>&arr){

    for(int i = 0;i< arr.size();i++){
        cout<<"a["<<i<<"]= ";
        cin>>arr[i];
        cout<<endl;
    }
}

void arr_output(vector<int>&arr){

    for(int i = 0;i< arr.size();i++)
        cout<<"a["<<i<<"]= "<<arr[i]<<endl;
}

int main(int args,char** argv){

    int n;

    cout<<"Enter the number of size of array: ";
    cin>>n;
    vector<int>arr(n,0);

    arr_input(arr);
    arr_output(arr);

    return 0;
}

File: lec_004(13th oct)/array.java
import java.util.Scanner;
public class array{
    public static void arr_input(int[] a){

        for(int i = 0;i< a.length;i++){
            System.out.print("a["+ i+ "]= ");
            a[i] = sc.nextInt();
            System.out.println();
        }
    }

    public static void arr_output(int[] a){

        for(int i = 0;i< a.length;i++){
            System.out.print("a["+ i+ "]= "+ a[i]);
            System.out.println();
        }
    }
    public static Scanner sc = new Scanner(System.in);
    public static void main(String[] args){

        System.out.print("Enter the size of array: ");
        int n = sc.nextInt();
        int[] a = new int[n];
        
        arr_input(a);
        arr_output(a);
    }
}